openstack/nova/templates/hypervisors/_ironic-deployment.yaml.tpl

```diff
@@ -24,17 +24,18 @@ spec:
   template:
     metadata:
       labels:
-{{ tuple . "nova" "compute" | include "helm-toolkit.snippets.kubernetes_metadata_labels" | indent 8 }}
+{{ tuple . "nova" $compute_hypervisor_name | include "helm-toolkit.snippets.kubernetes_metadata_labels" | indent 8 }}
```

Comment:
Could you leave it to "nova" and "compute". Otherwise it is hard to get all the compute nodes via labels

Reply:
This has been introduced by @galkindmitrii, please resolve this